processor/k8sattributesprocessor/config.go

```diff
@@ -166,6 +166,10 @@ type ExtractConfig struct {
 	// It is a list of FieldExtractConfig type. See FieldExtractConfig
 	// documentation for more details.
 	Labels []FieldExtractConfig `mapstructure:"labels"`
+
+	// OtelAnnotations extracts all pod annotations with the prefix "resource.opentelemetry.io" as resource attributes
+	// E.g. "resource.opentelemetry.io/foo" becomes "foo"
+	OtelAnnotations bool `mapstructure:"otel_annotations"`
```

Comment:
Can we also add a few words of text in the docs' README because right now the setting is only hidden here and in the configuration example of the README. Having an explicit section for this would make sense.